TemporaryOysters/bangazon-cli
# db structure update and payment type to db

##Pull Requests
This PR updates some missing fields in the database and also allows for adding payment options to database.


##Descriptions:

* Please describe what issue this PR solves
Adds the following attributes to Customers
  * email
  * changed status to 0/1 (boolean) instead of True/False

Adds the following attribute to PaymentOption
  * customerId

Created INSERT function to add PaymentOption to database
Created `return_payment_type` function to get PaymentOption from database


You will have to delete your bangazon.db and follow steps from the previous PR #37  to create and populate a new version. 







##Steps to Test:


  1. git fetch
  2. checkout the following branch me-bugfix
  3. Follow the steps from the PR listed above #37 
  4. cd into your `tests` folder
  5. run `python -m unittest discover -s . -p "Test*.py" -v`
  6. Confirm all tests are passing
##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?
None needed

* Any command line utilities to run?
See above
